### DISCUSSION TITLE
Internal mod, add a turn off switch to disable blueretro and use wired controllers is possible?

### DISCUSSION THREAD

**Discussion by BrunoMakesThings** [darthcloud/BlueRetro#298] [Q&A] [answered]
I'm intending to made some internal mods with this project, but i would like to keep the hability to use wired controllers.

Sorry if it is a double question, i've tried searching for it but didin't find the answer.

If i put a switch to turn off the 5v (3.3v or 8v, depending on the console), would it turn off the board, and with just that i would be able to use wired controllers, or would i need to switch other pins "off" as well?

Another question is, doing the same thing on a single port would i be abble to disable that single port, or is a turn all off or all on deal?
i'm new to this kind of mods. So i'm kind of lost.

Thanks!

**Comment by darthcloud** [comment, marked as answer]
You should be fine with a single switch on the power input as your say. This turn off the ESP32 completely.

Official internal mod is coming over the next few month, I suggest to wait for that.

**Reply by BrunoMakesThings** [reply]
Thanks! I'll keep my eye out for the official!
